cmd/identity/revocations.go

```diff
@@ -39,10 +38,6 @@ func init() {
 
 func cmdRevocations(cmd *cobra.Command, args []string) error {
 	ctx, _ := process.Ctx(cmd)
-	if len(args) > 0 {
-		revCfg.RevocationDBURL = "bolt://" + filepath.Join(configDir, args[0], "revocations.db")
```

Comment:
Can this be removed? @bryanchriswhite

Reply:
I'm pretty sure we need this for this command to work. What's the thinking here?

Reply:
I want to be able to pass in either a bolt or redis url. With the current setup, I can't pass in a redis url. I can modify it so it can use the above as a default for bolt if that's preferred.

Reply:
Ah, I see. So the idea here was to allow a use to pass a single argument that is the config subdirectory on which the bolt db lives. If you're using redis you wouldn't pass an argument, instead you would just pass the redis url in the flag. I think this can stay as is.

Reply:
oh you're right, adding that check back!